### PR TITLE
feat: Add functionality for adding people to a tag (GSoC)

### LINKED ISSUE #2552: Add support for adding people to tags (GSoC)
### Is your feature request related to a problem? Please describe.

Adding resolvers for adding people to tag functionality.

### Additional context

This is a companion issue of the one in the admin PalisadoesFoundation/talawa-admin#2302

### PR DESCRIPTION
**Pull request by meetulr**
### What kind of change does this PR introduce?

Adds functionality for adding multiple to tag.

### Issue Number:

Fixes #2552 

### Did you add tests for your changes?

Yes

### Additional Context:

Admin issue: PalisadoesFoundation/talawa-admin#2302

## Release Notes

- **New Features**
  - Introduced a mutation to assign multiple users to a tag, enhancing user tagging capabilities.
  - Added a resolver to retrieve users eligible for tagging, with support for filtering, sorting, and pagination.
  - Implemented a new input type for the mutation to streamline user tag assignments.
  - Added a new connection field to the UserTag type to view users not currently assigned to a tag.

- **Bug Fixes**
  - Implemented error handling for various scenarios, ensuring appropriate responses for invalid user or tag inputs.

- **Tests**
  - Added comprehensive unit tests for the new mutation and resolver, covering success and error scenarios.

### CODE REVIEW

File: src/resolvers/UserTag/usersToAssignTo.ts [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+      return {
+        _id: {
+          $lt: new Types.ObjectId(cursor),
+        },
+      };
+    }
+  } else {
+    return {};
+  }
+};
```
[the remark this answers is not in the document]
Comment (meetulr) [reply src/resolvers/UserTag/usersToAssignTo.ts:219]: cursor value will be an object id, this check is not required

**Comment by meetulr** [comment]
@coderabbitai all good?

**Review by rishav-jha-mech — APPROVED** [review]

**Review by beingnoble03 — APPROVED** [review]
@meetulr looks good to me. User tags are going to be super useful!

**Comment by meetulr** [comment]
@palisadoes please merge this so I can raise the next one.

**Pull request merged**